pkgs/by-name/ro/router/package.nix

```diff
@@ -39,6 +42,14 @@ rustPlatform.buildRustPackage rec {
     "-- --skip=query_planner::tests::missing_typename_and_fragments_in_requires"
   ];
 
+  passthru = {
+    updateScript = gitUpdater {
```

Comment:
Does not update cargoHash. I think you already fixed this in some of your other PRs :)